app/constants/channel.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export const MIN_CHANNELNAME_LENGTH = 2;
```

Comment:
there is a PR to change this to 1 character in v1 #6093 probably we want that here too

Reply:
Made a comment on the original PR that is also relevant here: Should we worry about the server version, or let the server fail?

Reply:
That is in fact a very very good question.. I would say if we can identify the error from the server and add a specific error string on the client for that I would let it fail, yes.

Reply:
The error string from the server seems good enough, so yes, we can let it fail.
![Screenshot from 2022-03-29 13-54-21](https://user-images.githubusercontent.com/1933730/160605489-45087309-6d29-46cd-8895-055e789a71be.png)